sem_covid/entrypoints/notebooks/rdf_transformer/rml_mapper_executor.py

```diff
@@ -0,0 +1,55 @@
+from abc import ABC, abstractmethod
+import json
+import requests as req
+from pathlib import Path
+from typing import List
+class RMLMapperABC(ABC):
+
+    @abstractmethod
+    def transform(self, rml_rule: str, sources: dict) -> str:
+        pass
+
+class RMLMapper(RMLMapperABC):
```

Comment:
We better document this class: what does it do and how shall it be used? 
Do we have passing tests for this class?